Dockerfile

```diff
@@ -9,8 +9,10 @@ RUN apt-get update && apt-get install -y \
     libgtest-dev \
     python3-pip \
     ssh \
-    sudo \
-    && apt-get clean \
+    sudo
+ARG ADDITIONAL_DEPENDENCIES
+RUN if [ "$ADDITIONAL_DEPENDENCIES" = "development" ]; then sudo apt install -y clangd clang-format; fi
```

Comment:
I think you can install that without the ARG since it won't clutter the final scratch image